test/testSetup.ts

```diff
@@ -113,23 +115,27 @@ export class TestSetup {
 
     // If you are not in a VSCode project, the Salesforce extensions are not running
     // Force the CLI integration extension to load before creating the project
-    const workbench = await (await browser.getWorkbench()).wait();
-    await utilities.runCommandFromCommandPrompt(workbench, 'Developer: Show Running Extensions', 5);
-    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Create Project', 5);
+    const workbench = await utilities.getWorkbench();
+    await utilities.showRunningExtensions();
+    const prompt = await workbench.executeQuickPick('SFDX: Create Project');
+    await utilities.waitForQuickPick(prompt, 'Standard', {
+      msg: 'Expected extension salesforcedx-core to be available within 5 seconds',
+      timeout: 5_000
+    });
```

Comment:
Waiting for the `Standard` button to appear is not needed to trigger the CLI Integration extension to activate.  Just running `SFDX: Create Project` from the command palette and escaping the command immediately after is sufficient.

Only when we actually want to create the project and need to click the `Standard` button do we need the `Standard` button to be present.

```suggestion
```

Reply:
@daphne-sfdc recall our want to establish correct state after an operation starts. This wait is an affirmation that the extension is installed and has started.